app/controllers/items_controller.rb

```diff
@@ -21,6 +21,7 @@ def create
     item.seller_id = current_user.id
     if item.valid?
       item.save
+      create_tags(item)
```

Comment:
The logic to create an item is now pretty complex. There might be other points in your app where you end up wanting to create an item. What if you pulled all of this into a command object called `CreateItem`?

```ruby
item = CreateItem.new(item_params, tag_names).call
```

where `tag_names` is:

```ruby
def tag_names
  params[:tags].split(",").map(&:strip)
end
```